alignak/objects/schedulingitem.py

```diff
@@ -2724,10 +2730,19 @@ def acknowledge_problem(self, notification_period, hosts, services, sticky, noti
             self.comments[comm.uuid] = comm
             self.broks.append(self.get_update_status_brok())
             self.raise_acknowledge_log_entry()
-            return comm
         else:
-            logger.warning("Acknowledge requested for %s %s but element state is OK/UP.",
-                           self.my_type, self.get_name())
+            logger.debug("Acknowledge requested for %s %s but element state is OK/UP.",
```

Comment:
If the host is in state OK, we don't want to acknowledge the services (lines after), I think in case we have a service in CRITICAL but the host UP, not want acknowledge them